requirements/edx-sandbox/base.in

```diff
@@ -16,14 +15,15 @@
 -c ../constraints.txt
 
 -r shared.txt                       # Dependencies in common with LMS and Studio
-chem                                # A helper library for chemistry calculations
+chem
 matplotlib==2.2.4                   # 2D plotting library
+networkx==2.2                       # Utilities for creating, manipulating, and studying network graphs
 numpy==1.16.5                       # Numeric array processing utilities; used by scipy
+openedx-calc
```

Comment:
@jmbowman @timmc-edx for calc, since it's using both in the lms and in the sandobx, I would assume we would want both to be the same versions even if that version is moving a bit.  Does it make more sense to pin this to `<2.0.0` so that we can get minor updates but not breaking ones?